server/services/map.js

```diff
@@ -50,42 +47,44 @@ module.exports = class MapService {
         }
 
         let isCustomGalaxy = game.settings.galaxy.galaxyType === 'custom';
+
+        // Get an array of random star names for however many stars we want.
+        const starNames = this.nameService.getRandomStarNames(isCustomGalaxy ? Math.min(playerLimit * 50, starLocations.length) : starCount);
```

Comment:
This is a code smell, we should not have to do this. The `starCount` should come into this function correct.

Assuming that the `starLocations.length` does get capped at `playerLimit * 50` there will be an issue further down in the function as there won't be enough star names to fulfil the entire the galaxy.